test/fixtures.py

```diff
@@ -0,0 +1,31 @@
+from shutil import copyfile
+
+import pytest
+from SCAutolib import models
+from pathlib import Path
+
+
+@pytest.fixture(scope="session")
+def backup_sssd_ca_db(tmp_path_factory):
+    backup = None
+    sssd_auth_ca_db = Path("/etc/sssd/pki/sssd_auth_ca_db.pem")
+    if sssd_auth_ca_db.exists():
+        # Save SSSD CA db
+        backup = tmp_path_factory.mktemp("backup").joinpath(
+            "sssd_auth_ca_db.pem")
+        copyfile(sssd_auth_ca_db, backup)
+
+    yield
+
+    # Restore SSSD CA db
+    if backup:
+        copyfile(backup, "/etc/sssd/pki/sssd_auth_ca_db.pem")
+    else:
+        sssd_auth_ca_db.unlink()
```

Comment:
Why do we remove the auth_ca_db file if we don't have backup?

Reply:
See this like 'leave the system after that in the same state as it was before test". Also, it would prevent the error if one cert would be used in several tests, but we would expect this. This kind of error is kind of hard to detetect. But we can discuss this more

Reply:
Yeah thats alright.